ms2query/app_helpers.py

```diff
@@ -178,7 +178,8 @@ def load_pickled_file(file_name: str):
     return contents
 
 
-def gather_zenodo_library(output_folder: str):
+def gather_zenodo_library(output_folder: str) \
+        -> Dict[str, Tuple[str, str, int]]:
```

Comment:
good solution! (also shows that we created some rather complicated output formats here...)